src/main/java/ch/wisv/areafiftylan/products/controller/TicketRestController.java

```diff
@@ -158,6 +160,24 @@ public AvailableTicketsDTO getAvailableTickets() {
         return createResponseEntity(HttpStatus.OK, "TicketType successfully deleted.");
     }
 
+    @PreAuthorize("hasRole('ADMIN')")
+    @PostMapping("/types/{typeId}/buyable")
+    public ResponseEntity<?> updateTicketTypeBuyable(@PathVariable Long typeId, @RequestBody @Validated TicketBuyableDTO buyable) {
+        TicketType ticketType = ticketService.getTicketTypeById(typeId);
+        ticketType.setBuyable(buyable.isBuyable());
+        ticketService.updateTicketType(typeId, ticketType);
```

Comment:
As you can see, the service allows you to update a whole type. Maybe let your endpoint reflect that functionality.